### PR TITLE
Fix/slow checkbox in assets

### PR DESCRIPTION
**Pull request by yenda**
There was two issues with the "Manage assets" screen when pressing a checkbox:
- it was taking extra time before subscriptions were recomputed
- it was taking extra time after as well

The first problem was apparently due to nested `touchable-highlight` components and fixed by making the checkbox a simple view
The second was caused by the re-rendering of every element of the list, sometimes multitple times, everytime the `:checked?` value of only one item was changed. To fix it I overwrote the `should-component-update` method.

### CODE REVIEW

File: src/status_im/ui/components/checkbox/view.cljs [hunk 1]
```diff
@@ -8,9 +8,9 @@
   "react/check-box is currently not available on iOS,
   use it once it is available on all platforms"
   [{:keys [on-value-change checked? accessibility-label
-           disabled? style icon-style]
+           disabled? style icon-style not-touchable?]
     :or   {accessibility-label :checkbox}}]
-  [react/touchable-highlight
+  [(if on-value-change react/touchable-highlight react/view)
```
Comment (flexsurfer) [inline src/status_im/ui/components/checkbox/view.cljs:13]: `not-touchable?` isn't used ?

**Review by flexsurfer — APPROVED** [review]

**Comment by rasom** [comment]
On my slow device it still takes 1.2-1.3s to check/uncheck token

**Comment by rasom** [comment]
@yenda it seem that after `:wallet.settings/toggle-visible-token` it takes 800ms till rendering of the particular checkbox begins, so I suppose we need to rearrange subs so that each item is subscribed separately or something like this.

**Comment by yenda** [comment]
@rasom but how long did it take before that? And are you sure that this is because of subscriptions? Did you check that they are ran immediately? 

Also are you talking about the numbers for this PR or develop?

**Comment by rasom** [comment]
@yenda this PR. Will check how much it takes to start subs calculations. Then will check develop

**Comment by rasom** [comment]
@yenda my mistake was that I checked dev env, in prod build it is faster:
 0.67-0.93s vs  1.7-2.3s in `develop`

**Review by rasom — APPROVED** [review]

**Pull request merged**